learn/indexing/optimize_indexing_performance.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: Optimize indexing performance by analyzing batch statistics 
+description: Learn how to analyze the `progressTrace` to identify and resolve indexing bottlenecks in Meilisearch.
+---
+
+# Optimize indexing performance by analyzing batch statistics
+
+Indexing performance can vary significantly depending on your dataset, index settings, and hardware. The [batch object](/reference/api/batches) provides information about the progress of asynchronous indexing operations.
+
+The `progressTrace` field within the batch object offers a detailed breakdown of where time is spent during the indexing process. By analyzing this data, you can identify bottlenecks and adjust configuration settings to improve indexing speed.
+
+## Understanding the `progressTrace`
+
+The `progressTrace` is a hierarchical trace showing each phase of indexing and how long it took.
+Each entry follows the structure:
+
+```json
+"processing tasks > indexing > extracting word proximity": "33.71s"
+```
+
+This means:
+
+- The step occurred during **indexing**.
+- The subtask was **extracting word proximity**.
+- It took **33.71 seconds**.
+
+Your goal is to focus on the **longest-running steps** and understand which index settings or data characteristics influence them.
+
+## Key phases and how to optimize them
+
+### Document processing
+
+| Trace key | Description | Optimization |
+|------------|--------------|--------------|
+| `computing document changes`, `extracting documents` | Meilisearch compares incoming documents to existing ones. | No direct optimization possible. The duration scales with the number and size of incoming documents.|
+
+### Filterable attributes
+
+| Trace key | Description | Optimization |
```

Comment:
What do you think about using a list instead of a single row table? Like what we do for the instance options: https://www.meilisearch.com/docs/learn/self_hosted/configure_meilisearch_at_launch#configuration-file-path

It's not super important and feel free to push back on this, but lists are easier to maintain than tables because of all the formatting markdown tables require.